apps/users/src/app/controllers/users.controller.v2.ts

```diff
@@ -87,12 +87,12 @@ export class UsersControllerV2 {
     @Query() userQuery?: ExpansionQuery
   ) {
     const user = isMe
-      ? authorizer.user
+      ? await authorizer.user
```

Comment:
```suggestion
      ? authorizer.user
```

ฝาก recheck หน่อยว่า มันเป็น User หรือ Promise<User> ถึงได้ await.